apps/web/src/components/Composer/NewPublication.tsx

```diff
@@ -541,7 +542,9 @@ const NewPublication: FC<Props> = ({ publication }) => {
             }
             onClick={createPublication}
           >
-            {isComment ? 'Comment' : 'Post'}
+            {isComment
+              ? t({ id: '[cta]Comment', message: 'Comment' })
```

Comment:
do we need cta prefix 🤔 ?

Reply:
Not right now actually. I added it for future compatibility. In the future, there may be many different "Comment" strings which need different translations depending on context, since "comment" can be both a verb and a noun.

Reply:
Gotcha 🚀

Reply:
This is also related to a bigger design question, about whether to use custom ID:s, or to use the source strings as ID.

Option A: https://lingui.js.org/tutorials/react-patterns.html#using-id-generated-from-message
Option B: https://lingui.js.org/tutorials/react-patterns.html#using-custom-id

With option A, you have something like id = "msg.welcome", with source text "Welcome to Lenster"
With option B, the the source string **is** the ID, so the id = "Welcome to Lenster".
Both have pros and cons.
Right now we use option B